doc/releasenotes.html

```diff
@@ -48,17 +54,205 @@ <h1 id="yyyy-mm-dd-r[n]">SAM yyyy.m.d [r[n]], SSC nnn: Month d, yyyy</h1>
+
+<h2>Detailed PV (SAM 2024.12.12)</h2>
+
+<p>Improve solar position algorithm to speed up PVWatts and Detailed PV simulation run times. (<a href="https://github.com/NREL/ssc/pull/1212">SSC 1212</a>)</p>
+
+<p>Update module and inverter libraries. (<a href="https://github.com/NREL/SAM/pull/1913">SAM 1913</a>, <a href="https://github.com/NREL/SAM/pull/1917">SAM 1917</a>)</p>
+
+<p>Rename temperature coefficient of power variable from 'gamma_r' to 'gamma_pmp' to be more consistent with industry standard. (<a href="https://github.com/NREL/ssc/pull/1231">SSC 1231</a>, <a href="https://github.com/NREL/SAM/pull/1898">SAM 1898</a>)</p>
+
+<p>Add subhourly clipping correction option based on matrix lookup approach ("Allen Method"). (<a href="https://github.com/NREL/SAM/pull/1861">SAM 1861</a>)</p>
+
+<p>Fix six-parameter module coefficient generator sanity check that caused a crash. This should result in more modules from the CEC list to be inlcuded in the SAM module library. (<a href="https://github.com/NREL/ssc/pull/1245">SSC 1245</a>)</p>
+
+<p>Fix solar position algorithm issue that caused SAM to crash when running multi-threaded such as for parametric simulations. (<a href="https://github.com/NREL/ssc/pull/1232">SSC 1232</a>)</p>
+
+<p>Fix implementation of NOCT cell temperature model. (<a href="https://github.com/NREL/ssc/pull/1233">SSC 1233</a>)</p>
+
+<p>Improve error messages for weather files with invalid data for SAM and PySAM. (<a href="https://github.com/NREL/ssc/pull/1219">SSC 1219</a>)</p>
+
+<p>Fix snow model for bifacial modules so that rear-side irradiance is not affected by snow cover. (<a href="https://github.com/NREL/ssc/pull/1214">SSC 1214</a>)</p>
+
+<h2>PVWatts (SAM 2024.12.12)</h2>
+
+<p>Change variable group name for 'losses' input variable from "Other DC losses" to "DC system losses". <a href ="https://github.com/NREL/ssc/pull/1229">SSC 1229</a>)</p>
+
+<p>Improve solar position algorithm to speed up PVWatts and Detailed PV simulation run times. (<a href="https://github.com/NREL/ssc/pull/1212">SSC 1212</a>)</p>
+
+<h2>Electric Battery Storage (SAM 2024.12.12)</h2>
+
+<p>Update and fix REopt API calls for battery size and dispatch optimization. (<a href="https://github.com/NREL/SAM/pull/1904">SAM 1904</a>, <a href="https://github.com/NREL/ssc/pull/1242">SSC 1242</a>, <a href="https://github.com/NREL/ssc/pull/1203">SSC 1203</a>)</p>
+
+<p>Add battery-specific availability loss inputs. (<a href="https://github.com/NREL/SAM/pull/1902">SAM 1902</a>, <a href="https://github.com/NREL/ssc/pull/1241">SSC 1241</a>, <a href="https://github.com/NREL/ssc/pull/1234">SSC 1234</a>)</p>
+
+<p>Modify battery dispatch to consider grid limits. (<a href="https://github.com/NREL/SAM/pull/1883">SAM 1883</a>, <a href="https://github.com/NREL/ssc/pull/1224">SSC 1224</a>)</p>
```

Comment:
Having this third feels a little like burying the lede on the battery changes. Mind putting it first?

Reply:
@brtietz I didn't consider order -- I'd welcome any other suggestions for putting things in order of importance.